hardware_interface:
  as:
    veh_interface:
      ros__parameters:
        json_filename: "veh_interface.json"
    speed_model:
      ros__parameters:
        json_filename: "speed_model.json"
    steering_model:
      ros__parameters:
        json_filename: "steering_model.json"
    veh_controller:
      ros__parameters:
        json_filename: "veh_controller.json"